src/entitySearchResultsList.js

```diff
@@ -98,7 +102,27 @@ export default class EntitySearchResultsList extends LightningElement {
         this.dispatchEvent(new CustomEvent('loadmore', {detail: {name: "juan"}}));
     }
 
-    handleSort(evt){
-        this.dispatchEvent(new CustomEvent('sort', {detail: {name: "juan"}}));
+    handleSort({ detail }) {
+        this.sortedBy = detail.fieldName;
+        this.sortDirection = detail.sortDirection;
```

Comment:
Normally, setting `this.sortedBy` and `this.sortDirection` on `lightning:datatable` should be enough to sort the table. I'm not sure why its not working, so we can just sort locally.

Reply:
Thanks